exes/ethercat/inc/public/tfc/ec.hpp

```diff
@@ -181,10 +181,9 @@ class context_t {
             i + 1);
         async_wait(true);
         return {};
-      } else {
-        context_.slavelist[0].state = EC_STATE_OPERATIONAL;
-        ecx::write_state(&context_, 0);
```

Comment:
This is being used, it is what happens while we wait for devices to reach operational.

Reply:
if stmt returns above

Reply:
It is in a loop. This path is taken often before the return is hit.

Reply:
Many devices especially our vfd's take a long time to transition to OPERATIONAL ethercat state. approx 2.7seconds